packages/www/components/Dashboard/AllSessionsTable/index.tsx

```diff
@@ -106,7 +106,7 @@ type SessionsTableData = {
 const AllSessionsTable = ({ title = "Sessions" }: { title?: string }) => {
   const { user, getStreamSessionsByUserId, deleteStream, deleteStreams } =
     useApi();
-  const tableProps = useTableState({ pageSize: 50 });
+  const tableProps = useTableState({ pageSize: 50, tableId: "sessions" });
```

Comment:
Out of curiosity, what are you using `tableId` for? I noticed its different than `queryKey`.

Reply:
we're using `tableId`. It was wrongly used in this PR. I fixed it now. `queryKey` is unnecessary, so I removed it